springdb/src/main/java/com/semis/gradvek/graphdb/CommandBuilder.java

```diff
@@ -79,6 +79,7 @@ private void appendEnabledDatasets(StringBuilder command) {
 
     private void appendTargetLimit(StringBuilder command) {
         command.append(" AND toUpper(nt.symbol) = '").append(target.toUpperCase(Locale.ROOT)).append("'");
+        // command.append(" AND nt.symbol = '").append(target.toUpperCase(Locale.ROOT)).append("'");
```

Comment:
what's this for?

Reply:
An extension to the performance test, that line would allow the query to utilize an index over the target symbol.  That would make the query an almost constant time operation rather than linear in the database size.  I didn't feel like deleting it in case I wanted to use it again, so I just commented it out.

Reply:
ok